apps/native/app/src/graphql/client.ts

```diff
@@ -134,6 +135,7 @@ const authLink = setContext(async (_, { headers }) => ({
     'X-Cognito-Token': `Bearer ${
       environmentStore.getState().cognito?.accessToken
     }`,
+    'User-Agent': getCustomUserAgent(),
```

Comment:
Do we want to add this also in the following places?:
* [In same file in errorLink ](https://github.com/island-is/island.is/blob/c3b59aac5837052c5f0b85355a6b6bff17f7250f/apps/native/app/src/graphql/client.ts#L80)
* [Here in auth-store?](https://github.com/island-is/island.is/blob/c3b59aac5837052c5f0b85355a6b6bff17f7250f/apps/native/app/src/stores/auth-store.ts#L123)